docs/REST.md

```diff
@@ -330,6 +331,28 @@ Upload new student information based on an excel binary string.
 }
 ```
 
+```
+## PATCH `/api/student/item`
+
+Edit an item's duedate that is checked out to a student
+
+### Parameters
+
+Query string:
+* `studentId`: Id of the student who has the item
+
+Body:
+* `itemAddress`: The address of the item
+* `date`: The new date the item is due
```

Comment:
1. There only other reference to a time is a models/items `timestamp` which is Unix epoch time.
2. I don't think it can parse an ole format considering it cannot convert 1486329601 (Unix epoch time) to the correct date: https://runkit.com/589360014664460014ad7254/589796829a353c0015f86697.